catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/converter/PostgreSqlTypeConverter.java

```diff
@@ -100,9 +107,34 @@ public String fromGravitinoType(Type type) {
       return BPCHAR + "(" + ((Types.FixedCharType) type).length() + ")";
     } else if (type instanceof Types.BinaryType) {
       return BYTEA;
+    } else if (type instanceof Types.ListType) {
+      return fromGravitinoArrayType((ListType) type);
     }
     throw new IllegalArgumentException(
         String.format(
             "Couldn't convert Gravitino type %s to PostgreSQL type", type.simpleString()));
   }
+
+  // PG doesn't support the multidimensional array internally. The current implementation does not
+  // enforce the declared number of dimensions either. Arrays of a particular element type are all
+  // considered to be of the same type, regardless of size or number of dimensions. So, declaring
+  // the array size or number of dimensions in CREATE TABLE is simply documentation; it does not
+  // affect run-time behavior.
+  // https://www.postgresql.org/docs/current/arrays.html#ARRAYS-DECLARATION
+  private String fromGravitinoArrayType(ListType listType) {
+    Type elementType = listType.elementType();
+    Preconditions.checkArgument(
+        !listType.elementNullable(), "PostgreSQL doesn't support element to nullable");
+    Preconditions.checkArgument(
+        !(elementType instanceof ListType),
```

Comment:
What if the `elementType` is a map or struct type?

Reply:
PG doesn't support map or struct

Reply:
fromGravitinoType will throw a exception for map or struct

Reply:
OK